packages/millicast-sdk/src/Publish.js

```diff
@@ -232,26 +238,27 @@ export default class Publish extends BaseWebRTC {
 
     const workerBlob = new Blob([workerString])
     const workerURL = URL.createObjectURL(workerBlob)
-    const worker = new Worker(workerURL)
+    this.worker = new Worker(workerURL)
```

Comment:
If we cannot ensure this.worker has been terminated here, should we check and terminate it?